src/styleguide/template.html

```diff
@@ -87,5 +87,75 @@
     </div></div>
 
     <script src="assets/js/app.js"></script>
+<script type="text/javascript">
+$(function() {
+    $(".color-block__background").each(function(){
```

Comment:
Classes (`.color-block...`) are declared in the CSS and so should only be used to apply a style in the HTML. So we use Data attributes (`[data-color-block...]`) to trigger a component from the JS.

Here, you should :
- Search for all the `[data-color-block-name]`, which _need_ to have as content the color code for the name they given.
- For each `[data-color-block-name]`, search for the associated color block which is provided by `[data-color-block]`, then read and convert its color.
- Write the found color code in `[data-color-block-name]`.

But you can also :
- Search for all `[data-color-block]` attributes, read and convert their colors, then save it in a `[ {name:..., code:...}, ... ]` array.
- Search for all `[data-color-block-name]` attributes and write inside the saved color.